### PR TITLE
Add new member permission setting info

### PR DESCRIPTION
**Pull request by lizokm**
Want this to be ready for EA.

### CODE REVIEW

File: docs/organization/membership/index.mdx [hunk 1]
```diff
@@ -115,6 +115,10 @@ Projects can only be accessed by that project's team(s). Team Admins, Org Manage
 
 All Org Members can get access to a project by joining a team associated with that project.
 
+### Adding New Members
+
+Users with Manager and Owner-level permissions can add, remove, and change members for a project or organization. They can also toggle on the "Let Members Invite Others" setting to allow any org member to freely invite anyone they like without needing org owner or manager approval.
```
Comment (sentaur-athena) [inline docs/organization/membership/index.mdx:120]: Should there be a mention that is EA only? I'm thinking this experiment will be EA for a while before going to GA.

**Review by sentaur-athena — APPROVED** [review]

File: docs/organization/membership/index.mdx (hunk 1 above)
Comment (lizokm) [reply docs/organization/membership/index.mdx:120]: Yes, great point. I'll add.

File: docs/organization/membership/index.mdx [hunk 2]
```diff
@@ -115,6 +115,10 @@ Projects can only be accessed by that project's team(s). Team Admins, Org Manage
 
 All Org Members can get access to a project by joining a team associated with that project.
 
+### Adding New Members
```
Comment (lizokm) [inline docs/organization/membership/index.mdx:118]:
```suggestion
### Adding New Members 
(Currently in [EA](/organization/early-adopter-features/).)
```

File: docs/organization/membership/index.mdx [hunk 3]
```diff
@@ -115,6 +115,11 @@ Projects can only be accessed by that project's team(s). Team Admins, Org Manage
 
 All Org Members can get access to a project by joining a team associated with that project.
 
+### Adding New Members 
+(Currently in [EA](/organization/early-adopter-features/).)
```
Comment (lizokm) [inline docs/organization/membership/index.mdx:119]:
```suggestion
<Note>
This feature is currently only available for [Early Adopters](/organization/early-adopter-features/).
</Note>
```

**Pull request merged**